test/publisher/topic_selector_test.exs

```diff
@@ -1,50 +1,125 @@
-defmodule KaufmannEx.Publisher.TopicSelectorTest do
+defmodule KaufmannEx.Publisher.Stage.TopicSelectorTest do
   use ExUnit.Case
-  alias KaufmannEx.Publisher.TopicSelector
+  alias KaufmannEx.Schemas.Event
+  alias KaufmannEx.Publisher.Request
```

Comment:
The alias `KaufmannEx.Publisher.Request` is not alphabetically ordered among its group.